### DISCUSSION TITLE
Is there any documentation on what the JDM graphs look like?

### DISCUSSION THREAD

**Discussion by scoobaSteve007** [gorules/zen#158] [Q&A]
Our organization is not ready to use the fully built JDM editor provided. Are there any examples anywhere that show what the graph files look like? Or is the only way currently to create rules is via the JDM editor.

**Comment by ivanmiletic** [comment]
Hi @scoobaSteve007 at the moment we do not have it documented, you can see the models by downloading JSON from editor. We will release initial models in around 3-4 months as we are reviewing adding one more built in node.

**Comment by fung** [comment]
Thanks, this will be very helpful for us.
